switching/input/messages/R1.py

```diff
@@ -183,6 +183,22 @@ def get_type_from_subtype(self, subtype):
                 return x['type']
         return []
 
+    def get_minimum_fields(self, subtype):
```

Comment:
Change this method to use the `subtype` of the instance `self` because if we want the minimum fields of an R1 instance we only want to call `r1.get_minium_fields()` because the instance already has de `subtype`.